tests/modules/segmentation/graph_utils/test_edge_attributes.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+import h5py
+import numpy as np
+import networkx as nx
+from pathlib import Path
+
+from ophys_etl.modules.segmentation.graph_utils import (
+        creation, edge_attributes)
+
+
+@pytest.fixture
+def video_path(tmpdir, request):
+    vpath = Path(tmpdir / "video.h5")
+    data = np.random.randint(0, 2**15,
```

Comment:
Are there any tests that use static data to ensure that calculations like Pearson are being performed correctly?

Reply:
There are not. There could be relatively easily for Pearson, comparing `1.0 - cdist( , , metric="correlation")` to a `pearsonr()` call, but, I didn't see a lot of value in confirming that.
The other ones would be more complicated to reproduce the value without just writing the same functionality again, perhaps in a different way, to test against.
Probably these could be improved to separate out the filtering portions, to test those independently.
I don't see much value in static data tests here.

Reply:
It sounds like most of these are just 'smoke' tests then, it would valuable to have actual numeric tests.

Reply:
Based on the request, I am just imagining a snapshot of returns from these methods now, and then just checking that we match these snapshots in the future. The purpose would be to check for changes from scipy changes or something

Reply:
added some basic numerical tests